guides/v2.1/config-guide/multi-site/ms_nginx.md

```diff
@@ -51,11 +52,54 @@ Setting up multiple stores consists of the following tasks:
     *   `$MAGE_RUN_CODE` is the unique website or store view code that corresponds to `$MAGE_RUN_TYPE`.
 
 ## Step 2: Create nginx virtual hosts {#ms-nginx-vhosts}
-This section discusses how to load websites on the {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %}. You can use either websites or store views; if you use store views, you must adjust parameter values accordingly. You must complete the tasks in this section as a user with `root` privileges.
+This section discusses how to load websites on the {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %}. You can use either websites or store views; if you use store views, you must adjust parameter values accordingly. You must complete the tasks in this section as a user with at least `sudo` privileges.
 
-{% collapsible To create virtual hosts: %}
+You can use one [nginx virtual host](#ms-nginx-vhosts) or many [nginx virtual host](#ms-nginx-vhosts) to load multiple websites. Using just one virtual host file allows you too keep your [nginx virtual host](#ms-nginx-vhosts) file simple and clean, and using many [nginx virtual host](#ms-nginx-vhosts) allows customizations for each particular store, such as a custom location for `french.mysite.mg`.
 
-1.	Open a text editor and add the following contents to a new file named `/etc/nginx/sites-available/french.mysite.mg.conf`:
+{% collapsible To use one virtual host (simplified): %}
+
+This configuration expands upon [Magento Nginx Configuration]({{ page.baseurl }}/install-gde/prereq/nginx.html). To create one virtual host: 
+
+1.	Open a text editor and add the following contents to a new file named `/etc/nginx/sites-available/magento`:
+    ```
+    map $http_host $MAGE_RUN_CODE {
+        default '';
+        french.mysite.mg french;
+        german.mysite.mg german;
+    }
+
+    server {
+        listen 80;
+        server_name mysite.mg french.mysite.mg german.mysite.mg;
+        set $MAGE_ROOT /var/www/html/magento2;
+        set $MAGE_MODE developer;
+        set $MAGE_RUN_TYPE website; #or set $MAGE_RUN_TYPE store;
+        include /var/www/html/magento2/nginx.conf;
+    }
+    ```
+2.	Save your changes to the files and exit the text editor.
+3.	Verify the server configuration:
+
+		nginx -t
+4.	If successful, the following message displays:
+
+		nginx: configuration file /etc/nginx/nginx.conf test is successful
+
+	If errors display, check the syntax of your virtual host configuration files.
+
+5.	Create symbolic link in the `/etc/nginx/sites-enabled` directory:
+
+		cd /etc/nginx/sites-enabled
+		ln -s /etc/nginx/sites-available/magento magento
+
+For more detail about the map directive, see [nginx documentation on the map directive](http://nginx.org/en/docs/http/ngx_http_map_module.html#map){:target="_blank"}.
```

Comment:
For more details (plural)

Reply:
@shrielenee it should show CLA as signed now.